docs/src/kernel/advanced-instructions.md

```diff
@@ -2,26 +2,30 @@
 
 ## User-Mode Unit Tests
 
-Asterinas consists of many crates, some of which do not require a VM environment and can be tested with the standard `cargo test`. They are listed in the root `Makefile` and can be tested together through the following Make command.
+Asterinas consists of many crates,
+some of which do not require a VM environment and can be tested with the standard `cargo test`.
+They are listed in the root `Makefile`
+and can be tested together through the following Make command.
 
 ```bash
-make test
+/asterinas$ make test
 ```
 
 To test an individual crate, enter the directory of the crate and invoke `cargo test`.
 
 ### Kernel-Mode Unit Tests
 
-Many crates in Asterinas do require a VM environment to be tested. The unit tests for these crates are empowered by our [ktest](framework/libs/ktest) framework.
+Many crates in Asterinas do require a VM environment to be tested.
+The unit tests for these crates are empowered by OSDK.
 
 ```bash
-make run KTEST=1
+/asterinas$ make ktest
```

Comment:
I know what you mean by `<path>$ <command>`. But this syntax is non-standard. And for most of the time, the path is simply the root of the project. Thus, it is more of a distraction than an assistance. So I suggest removing `<path>$` from the commands and write in plain English to emphasize when the current working directory should be changed.